tests/integration/telemetry/outboundtrafficpolicy/helper.go

```diff
@@ -321,7 +325,7 @@ func setupEcho(t *testing.T, ctx resource.Context, mode TrafficPolicy) (echo.Ins
 		With(&dest, echo.Config{
 			Service:   "destination",
 			Namespace: appsNamespace,
-			Subsets:   []echo.SubsetConfig{{}},
+			Subsets:   []echo.SubsetConfig{{Annotations: echo.NewAnnotations().SetBool(echo.SidecarInject, false)}},
```

Comment:
This is more accurate to simulating external traffic as we don't rely on envoy->envoy doing things that external services won't do